packages/expo-updates/ios/EXUpdates/Update/EXUpdatesLegacyUpdate.m

```diff
@@ -133,6 +133,9 @@ + (NSURL *)bundledAssetBaseUrlWithManifest:(NSDictionary *)manifest config:(EXUp
     if (maybeAssetsUrl && maybeAssetsUrl.scheme) {
       return maybeAssetsUrl;
     } else {
+      if ([assetsPathOrUrl hasPrefix:@"./"]) {
+        assetsPathOrUrl = [assetsPathOrUrl substringFromIndex:2];
+      }
```

Comment:
Replace this with the more robust https://developer.apple.com/documentation/foundation/nsurl/1411073-standardizedurl?language=objc